src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+#Disable Spring Boot Banner
+spring.main.banner-mode=off
+
+#Exposing "/health" endpoint on base-path ="/"
```

Comment:
We can show the banner in logs, so let's remove the property. Also, let's remove comments as well.